pkg/squashfs/squashfs.go

```diff
@@ -135,7 +155,6 @@ func (s *Snap) Build(buildDir string) error {
 		return runCommand(
 			"mksquashfs",
 			".", fullSnapPath,
-			"-all-root",
```

Comment:
Uhhh, sorry for this. This looks like an accident, there is https://github.com/ubuntu-core/snappy/pull/126 for this. let me see why its there.